spec/support/shared_examples/service_pattern_examples.rb

```diff
@@ -0,0 +1,27 @@
+RSpec.shared_examples "ServicePatternExamples" do
+  describe "#call" do
+    context "when the #call method is not implemented" do
+      let(:test_class) { Class.new { include ServicePattern } }
+
+      it "raises a NotImplementedError" do
+        expect { test_class.call }.to raise_error(NoMethodError, "#call must be implemented")
+      end
+    end
```

Comment:
I've specifically used a `test_class` here because all of our service objects implement `#call` which is how this went untested in the first place. I believe that this is a valid test.

However, you have identified that we should _also_ be testting what happens when the service object does implement the `#call` method.

Reply:
As discussed:

1. Move the first context into its own file: `spec/services/concerns/service_pattern_spec.rb`
2. Test the instance method `#call` rather than the class method `.call`.